sphinx_search/extension.py

```diff
@@ -1,26 +1,43 @@
 import os
+
 from sphinx.util.fileutil import copy_asset
 
 
-CUSTOM_ASSETS_FILES = [
-    os.path.join('js', 'rtd_sphinx_search.min.js'),
-    os.path.join('css', 'rtd_sphinx_search.min.css'),
-]
+CUSTOM_ASSETS_FILES = {
+    'MINIFIED': [
+        os.path.join('js', 'rtd_sphinx_search.min.js'),
+        os.path.join('css', 'rtd_sphinx_search.min.css'),
+    ],
+    'UN_MINIFIED': [
+        os.path.join('js', 'rtd_sphinx_search.js'),
+        os.path.join('css', 'rtd_sphinx_search.css'),
+    ]
+}
 
 
 def copy_asset_files(app, exception):
     if exception is None:  # build succeeded
-        for f in CUSTOM_ASSETS_FILES:
-            path = os.path.join(os.path.dirname(__file__), '_static', f)
-            copy_asset(path, os.path.join(app.outdir, '_static', f.split('.')[-1]))
+        files = CUSTOM_ASSETS_FILES['MINIFIED'] + CUSTOM_ASSETS_FILES['UN_MINIFIED']
+        for file in files:
+            path = os.path.join(os.path.dirname(__file__), '_static', file)
+            copy_asset(path, os.path.join(app.outdir, '_static', file.split('.')[-1]))
 
 
-def setup(app):
+def inject_static_files(app):
+    """Inject correct CSS and JS files based on the value of ``RTD_SPHINX_SEARCH_FILE_TYPE``.""" 
+    file_type = app.config.RTD_SPHINX_SEARCH_FILE_TYPE
+    files = CUSTOM_ASSETS_FILES[file_type]
 
-    app.connect('build-finished', copy_asset_files)
-
-    for file in CUSTOM_ASSETS_FILES:
-        if file.endswith('.min.js'):
+    for file in files:
+        if file.endswith('.js'):
             app.add_js_file(file)
-        if file.endswith('.min.css'):
+        elif file.endswith('.css'):
             app.add_css_file(file)
+
+
+def setup(app):
+
+    app.add_config_value('RTD_SPHINX_SEARCH_FILE_TYPE', 'UN_MINIFIED', 'html')
```

Comment:
I believe Sphinx uses lower-case config values by convention.

Reply:
This also should be documented.

Reply:
Can the documentation changes be done in another PR with the API reference?